pr-data.csv

```diff
@@ -1940,8 +1940,8 @@ https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop
 https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app,org.apache.hadoop.mapreduce.jobhistory.TestJobHistoryEventHandler.testSigTermedFunctionality,NIO,Accepted,https://github.com/apache/hadoop/pull/2499,
 https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-app,org.apache.hadoop.mapreduce.jobhistory.TestJobHistoryEventHandler.testUnflushedTimer,OD,,,
 https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core,org.apache.hadoop.mapreduce.lib.db.TestSplitters.testBooleanSplitter,OD,,,
-https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.mapreduce.lib.input.TestCombineFileInputFormat.testSplitPlacement,ID,,,
-https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.mapreduce.lib.input.TestCombineFileInputFormat.testSplitPlacementForCompressedFiles,ID,,,
+https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.mapreduce.lib.input.TestCombineFileInputFormat.testSplitPlacement,ID,Opened,https://github.com/apache/hadoop/pull/8010,
+https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.mapreduce.lib.input.TestCombineFileInputFormat.testSplitPlacementForCompressedFiles,ID,DeveloperFixed,,could not reproduce: https://github.com/apache/hadoop/commit/551615fa13f65ae996bae9c1bacff189539b6557
```

Comment:
You cannot claim `DeveloperFixed` unless you can find where it was fixed. Did you check that the test was `ID` in the commit where it was reported (`cc2babc1f75c93bf89a8f10da525f944c15d02ea`)? Did you run the entire test class (or even the entire module) in the latest commit?